.github/workflows/codegen-ci.yml

```diff
@@ -32,13 +32,32 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - name: Repo Checkout
+        uses: actions/checkout@v2
+
```

Comment:
does moving this up affect the responsiveness of canceling previous runs? Just wondering why it was moved

Reply:
The repo checkout can overwrite some of the previous work that was done. I ran into a weird situation with the python-ci.yml where not having the Repo Checkout first caused the gcloud auth step to fail. So I decided to do it first for all of them just to make sure there wasn't a race condition.

Reply:
Thanks. Sounds good